mrp_operations_extension/models/mrp_production.py

```diff
@@ -81,8 +64,9 @@ def _compute_possible_workcenters(self):
     do_production = fields.Boolean(string='Produce here')
     time_start = fields.Float(string="Time Start")
     time_stop = fields.Float(string="Time Stop")
-    move_lines = fields.One2many(
-        comodel_name='stock.move', inverse_name='work_order', string='Moves')
+    move_lines = fields.Many2many(
+        'stock.move', compute="_compute_move_lines", string='Moves',
```

Comment:
The parameter name can be omitted as we are declaring the parameters in the same order as the declaration, it isn't necessary at all... it saves line width